dashboard/src/components/ServiceInstanceView/ServiceInstanceView.tsx

```diff
@@ -162,15 +162,44 @@ class ServiceInstanceView extends React.Component<IServiceInstanceViewProps> {
         planCard = this.renderSVCPlan(svcPlan);
       }
 
-      const bindings =
-        instance &&
-        bindingsWithSecrets.list.filter(
-          b =>
-            b.binding.spec.instanceRef.name === name && b.binding.metadata.namespace === namespace,
+      if (svcClass && svcClass.spec.bindable) {
+        const bindings =
+          instance &&
+          bindingsWithSecrets.list.filter(
+            b =>
+              b.binding.spec.instanceRef.name === name &&
+              b.binding.metadata.namespace === namespace,
+          );
+        bindingSection = (
+          <div>
+            <h2>Bindings</h2>
+            <AddBindingButton
+              bindingSchema={svcPlan && svcPlan.spec.serviceBindingCreateParameterSchema}
+              instanceRefName={instance.metadata.name}
+              namespace={instance.metadata.namespace}
+              addBinding={this.props.addBinding}
+              onAddBinding={this.onAddBinding}
+              error={this.props.errors.create}
+            />
+            <br />
+            {this.props.errors.delete && (
+              <ErrorSelector
+                error={this.props.errors.delete}
+                resource="Binding"
+                action="delete"
+                defaultRequiredRBACRoles={RequiredRBACRoles}
+              />
+            )}
+            <BindingList bindingsWithSecrets={bindings} removeBinding={this.props.removeBinding} />
+          </div>
         );
-      bindingList = (
-        <BindingList bindingsWithSecrets={bindings} removeBinding={this.props.removeBinding} />
-      );
+      } else {
+        bindingSection = (
+          <h5 className="padding-b-big">
```

Comment:
I would keep the `<h2>Bindings</h2>` heading, and then just add a paragraph with a message like "This Instance cannot be bound to applications."